pacai/util/probability.py

```diff
@@ -94,7 +90,7 @@ def chooseFromDistribution(distribution):
     Takes either a counter or a list of (prob, key) pairs and samples.
```

Comment:
Isn't this just `sample`?
Can we git rid of this?

Reply:
The ABC for ghost agent calls this method in the getAction, it passes a dictionary with a probability distribution for possible actions. We can get rid of this method call in the ABC and call sample() instead.

Reply:
Feel free to change the ghost if we can just use this.

Reply:
Ok.